### ISSUE TITLE
Skill: 94 funny personal questions

### ISSUE THREAD

**Issue by tabesin** [fossasia/susi.ai#2567]
**Actual Behaviour**

Here are 94 questions Cortana could handle
https://www.digitalcitizen.life/top-50-most-funny-cortana-commands-questions

Susi does not implement them. but worse, APIs which Susi accesses implement them:

- Who is your father?
- I was created by Stephen Wolfram and his team.

**Expected Behaviour**

Susi needs to have elaborate answers for all of these personal questions.

**Would you like to work on the issue?**

**Comment by Jayashree-panda** [comment]
@mariobehling @tabesin Can I take this issue?

**Comment by tabesin** [comment]
Yes, this us very much appreciated but feel free to divide it in subtasks.

**Comment by Jayashree-panda** [comment]
@tabesin Yeah! Sure, I will.
Thank:)

**Comment by PratishShrivastava** [comment]
**Can i take this issue.**

**Comment by PratishShrivastava** [comment]
**@tabesin** Did I have to add 94 questions for Susi like Cortana has
